app/controllers/corporate_categories_controller.rb

```diff
@@ -20,15 +19,12 @@ def paywall_debt_advice_evaluation_toolkit
     end
   end
 
-  def article_interactor
-    Core::CorporateReader.new(default_article_id)
+  def article_resource
+    Mas::Cms::Article.find(default_article_id, locale: I18n.locale)
```

Comment:
Not sure if this was intended but before this PR, the code was trying to find Corporate pages (using Core::CorporateReader) and now it is replaced for Article pages. Is that what is intended? I am asking because for corporate article there is on the gem the class `Mas::Cms::Corporate`.